test/modules/layers/test_quantization.py

```diff
@@ -84,23 +88,72 @@ def test_preprocess(self):
         actual_flat_shape = torch.tensor(encoded_flat.shape)
         actual_permuted_shape = torch.tensor(permuted_shape)
 
-        assert torch.equal(
-            actual_flat_shape, expected_flat_shape
-        ), f"actual flattened shape: {actual_flat_shape}, expected flattened shape: {expected_flat_shape}"
+        assert_expected(actual_flat_shape, expected_flat_shape)
 
-        assert torch.equal(
-            actual_permuted_shape, expected_permuted_shape
-        ), f"actual permuted shape: {actual_permuted_shape}, expected permuted shape: {expected_permuted_shape}"
+        assert_expected(actual_permuted_shape, expected_permuted_shape)
 
     def test_preprocess_channel_dim_assertion(self):
         with self.assertRaises(ValueError):
-            encoded_flat, permuted_shape = self.vq._preprocess(self.encoded[:, :4, :])
+            _, _ = self.vq._preprocess(self.encoded[:, :4, :])
 
     def test_postprocess(self):
         quantized = self.vq._postprocess(self.input_tensor_flat, torch.Size([2, 2, 3]))
         actual_quantized_shape = torch.tensor(quantized.shape)
         expected_quantized_shape = torch.tensor([2, 3, 2])
 
-        assert torch.equal(
-            actual_quantized_shape, expected_quantized_shape
-        ), f"actual quantized shape: {actual_quantized_shape}, expected quantized shape: {expected_quantized_shape}"
+        assert_expected(actual_quantized_shape, expected_quantized_shape)
+
+    def test_embed_init(self):
```

Comment:
More explicit this way: `test_{the actual method name}` instead of shortening it